lib/splash.js

```diff
@@ -47,24 +46,25 @@ function logObject(configuration) {
 }
 
 function valueToShow(key, value) {
-    var hideValue = _.reduce(KEYS_TO_HIDE, function(memo, keyToHide) {
-        var regExp = new RegExp(".*" + keyToHide + ".*", "i");
-        return memo || (key.match(regExp) !== null);
-    }, false);
 
-    return hideValue ? "****" : value;
+    const isKeyASecret = (memo, keyToHide) => {
+        // Does the key match one of the hidden keys pattern
+        return memo || (key.match(new RegExp(`.*${keyToHide}.*`, "i")) !== null);
+    };
+
+    return _.reduce(KEYS_TO_HIDE, isKeyASecret, false) ? "****" : value;
```

Comment:
I think `_.some()` is a better option than `_.reduce()` for what we wanna do here.
But as `KEYS_TO_HIDE` only contains 2 elements, this remarks is less than "minor" :)